### ISSUE TITLE
[Feature] Allow users to write markdown within the table widget 

### ISSUE THREAD

**Issue by abhvsn** [appsmithorg/appsmith#6661]
## Summary

Currently, in Appsmith, there is limited functionality to update the table columns which consists of text alignment, text color, cell background etc. Users might want to have the custom styling as per their needs.   

User on the discord requested for this feature : 

> Hello, is there a way to insert custom markdown into a table column (cell)? I'd like to customize the table display, like showing a star rating column.

**Comment by somangshu** [comment]
@abhvsn we could add rate column using the rating widget to the table, Adding markdown / HTML here might have a technical challenge in implementation right now

**Comment by abhvsn** [comment]
@somangshu This will fulfill the requirement for the current user, but in future we should address this request as users will have the freedom of custom styling the columns as per their will.

Also is there an issue where we are maintaining the widget list which Appsmith is working on to integrate with table widget, can you please add the rating widget to that list 🙂

**Comment by somangshu** [comment]
Yes #2860 is where all tasks are right now, This we will segregated and kicked off soon

Regarding the html / MD in table, I think it should be possible once we achieve #2308

**Comment by robvandyck** [comment]
As a real-world example: I'm reading in search results from Elasticsearch. The results have highlighting included.
I was trying to use the highlighting in one table column using the following 'Computed Value':
`{{ 
  function() {
         const str = currentRow.highlight.message[0];
	 var newStr = str.replace(new RegExp('@kibana-highlighted-field@', 'g'), '<b>').replace(new RegExp('@/kibana-highlighted-field@', 'g'), '</b>');
	 return newStr;
   }()
}}`

But this results in actual '<b>' and '</b>' tags within the search results, instead of the expected boldness.

**Comment by pranavkanade** [comment]
@dilippitchika

**Comment by dilippitchika** [comment]
Hey @robvandyck thanks for adding more context on this, it will help us prioritise this better.
